bin/cml/pr/create.js

```diff
@@ -0,0 +1,76 @@
+const kebabcaseKeys = require('kebabcase-keys');
+
+const {
+  GIT_REMOTE,
+  GIT_USER_NAME,
+  GIT_USER_EMAIL
+} = require('../../../src/cml');
+
+exports.command = 'create <glob path...>';
```

Comment:
/note to self `cml pr update` https://github.com/iterative/cml/issues/1126 ?

Reply:
Probably, but not on this pull request 😅

Reply:
> Probably, but not on this pull request

agreed

Reply:
Resolving in favor of https://github.com/iterative/cml/issues/1126#issuecomment-1222727199